src/main/java/teammates/ui/controller/InstructorFeedbackQuestionEditAction.java

```diff
@@ -68,12 +67,24 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                                             .getInstructorFeedbackEditLink(courseId, feedbackSessionName));
     }
 
-    private void deleteQuestion(FeedbackQuestionAttributes updatedQuestion) {
-        logic.deleteFeedbackQuestion(updatedQuestion.getId());
+    private void deleteQuestion() {
+        String questionId = getRequestParamValue(Const.ParamsNames.FEEDBACK_QUESTION_ID);
+        Assumption.assertPostParamNotNull(Const.ParamsNames.FEEDBACK_QUESTION_ID, questionId);
```

Comment:
Can use `getNonNullRequestParamValue` instead

Reply:
Hmm I see at least 10 places in this file can use this change. Should I change them all?

Reply:
Yup!!